lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<-EOM.gsub(/^ {6}/, '')
+
+      '#{METADATA_REPOSITORY}' repository not found.
+      Try running the command:
+        git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git "#{metadata_dir}"
```

Comment:
> We could peek at their .git/config file and find out and report accordingly. How their 'origin' reference is formed, we will know what their preference is.

That's a lot of functionality that is not really the responsibility of the generate class.
If someone wants to make another module with this functionality I'd be happy to use it, but it seems like a lot of work for a single line of an error message which hopefully no-one will see anyway.

> there is value to being inclusive, not excluding those that prefer another form.

It's not excluding anybody.

The proposed change is an improvement over the original mystery error message:
```
[snip..]/xruby/lib/generator.rb:19:in `read': No such file or directory @ rb_sysopen - [snip..]/x-common/leap.json (Errno::ENOENT)
[snip stack trace]
```
And provides some immediately actionable clues about what might need to be done to fix the error. The user still has the option to type a different command into their command line, modify the suggested line in any way, or to seek out other information.

Is this a case of "The Perfect being the enemy of the Good"?